src/models/product-variant-model.js

```diff
@@ -109,6 +109,11 @@ const ProductVariantModel = BaseModel.extend({
     * @type {Array}
   */
   get imageVariants() {
+    const image = this.image;
+
+    if (typeof image === 'undefined') {
```

Comment:
Just use `if (!image)`. Don't bind to an explicit falsey type. Just let it be falsey.